src/script/page/preferences/Preferences.tsx

```diff
@@ -88,42 +105,28 @@ const Preferences: React.FC<PreferencesProps> = ({listViewModel, contentViewMode
   ];
 
   return (
-    <Transition in={isVisible} timeout={300}>
-      <div
-        id="preferences"
-        className={`left-list left-list-preferences ${isVisible && 'left-list-is-visible'}`}
-        aria-hidden={isVisible ? 'false' : 'true'}
-      >
-        <div className="left-list-header">
-          <span className="left-list-header-text">{t('preferencesHeadline')}</span>
-          <button
-            type="button"
-            className="left-list-header-close-button button-icon-large"
-            onClick={close}
-            title={t('tooltipSearchClose')}
-            data-uie-name="do-close-preferences"
-          >
-            <Icon.Close />
-          </button>
-        </div>
-        <div className="left-list-center">
-          <ul className="left-list-items preferences-list-items">
-            {items
-              .filter(item => !item.hidden)
-              .map(item => (
-                <PreferenceItem
-                  key={item.id}
-                  label={item.label}
-                  onSelect={() => contentViewModel.switchContent(item.id)}
-                  isSelected={contentState === item.id}
-                  uieName={item.uieName}
-                  IconComponent={item.IconComponent}
-                />
-              ))}
-          </ul>
-        </div>
-      </div>
-    </Transition>
+    <LeftListWrapper
```

Comment:
The wrapping elements for all the list on the left of the app are common,
So I extracted the left list logic to put it in the wrapper. We can use this component when migrating more lists to React